js/src/agile-toolkit-ui.js

```diff
@@ -1,5 +1,5 @@
 import 'core-js/stable';
-import atkSemantic from 'atk-semantic-ui';
+import atkFomantic from 'atk-fomantic-ui';
```

Comment:
not sure if this rename exactly, but unminified size has changed by -50% probably by this PR:

- before https://github.com/atk4/ui/blob/d7bdc09dceba2e8575ff0ae39e9ee12ab60e9781/public/atkjs-ui.js
- after https://github.com/atk4/ui/blob/80b1e9b23322681b8203f4faf7f09d4a182f1fb3/public/atkjs-ui.js